savaslabs/savaslabs.github.io
# Issue #2675: multiple blog authors

@kostajh This PR does the following, assuming blog post `author` properties follow the following convention:

`author: Anne Tomasevich`
`author: Anne Tomasevich and Ro Wang`
`author: Anne Tomasevich, Ro Wang, and Kosta Harlan`

(basically a normal English phrase)

- Updates the logic at the top of blog posts to handle multiple authors. Current team members' names will be linked to their team page; former team members' names will not be linked.
- Updates the "About the author" section to:
  - Have the heading "About the authors" if there are multiple authors
  - Show the image/description for each author
  - If the author is a current team member, link their image to their team page
- Updates the `post-bottom` section to look a little better on various screen sizes
- Refactors some SCSS to only include what's needed in `critical.css`
- Refactors some HTML to improve markup and make filenames more accurate/intuitive

**To test:**

- Update the `author` property of a blog post to include multiple authors.
- Check out the following places where author names exist:
  - Blog cards on the front page
  - Blog cards on /blog
  - Blog cards on team member pages
  - Top of blog posts
  - Bottom of blog posts

**Notes:**
- I do like your idea of including author images up top, but I'd prefer that it be a separate issue.
- One shortcoming of this work is that, in the "About the authors" section, authors will always be ordered by
  - Current team members followed by former team members (I think this is ok)
  - Alphabetical by first name (this is less optimal and clearly the author of this code benefits the most!)

Let me know if you have any suggestions or if I missed anything!
